spec/support/shared_contexts/cli/helpers/when_there_are_existing_subgroups_context.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+shared_context 'when there are subgroups which are belong to selected owner' do
```

Comment:
I'd go with a [shared example](https://relishapp.com/rspec/rspec-core/v/3-3/docs/example-groups/shared-examples) to inject the  `default_performer` and any other `lets` that comes from external stuff.
```suggestion
shared_example 'subgroups that belong to' do |default_owner: , selected_group_index: |
```
And later we can use it 
```ruby
it_should_behave_like 'subgroups that belong to', default_owner: owner, selected_group_index: 2
```